tap-agent/src/agent/sender_allocation.rs

```diff
@@ -696,31 +704,43 @@ impl SenderAllocationState {
                     error!("Failed to recover receipt signer: {}", e);
                     anyhow!(e)
                 })?;
-            sqlx::query!(
-                r#"
-                    INSERT INTO scalar_tap_receipts_invalid (
-                        signer_address,
-                        signature,
-                        allocation_id,
-                        timestamp_ns,
-                        nonce,
-                        value,
-                        error_log
-                    )
-                    VALUES ($1, $2, $3, $4, $5, $6, $7)
-                "#,
-                receipt_signer.encode_hex(),
-                encoded_signature,
-                allocation_id.encode_hex(),
-                BigDecimal::from(receipt.message.timestamp_ns),
-                BigDecimal::from(receipt.message.nonce),
-                BigDecimal::from(BigInt::from(receipt.message.value)),
-                receipt_error
-            )
-            .execute(&self.pgpool)
-            .await
-            .map_err(|e| anyhow!("Failed to store invalid receipt: {:?}", e))?;
+            reciepts_signers.push(receipt_signer.encode_hex());
+            encoded_signatures.push(encoded_signature);
+            allocation_ids.push(allocation_id.encode_hex());
+            timestamps.push(BigDecimal::from(receipt.message.timestamp_ns));
+            nounces.push(BigDecimal::from(receipt.message.nonce));
+            values.push(BigDecimal::from(BigInt::from(receipt.message.value)));
         }
+        sqlx::query!(
+            r#"INSERT INTO scalar_tap_receipts_invalid (
+                signer_address,
+                signature,
+                allocation_id,
+                timestamp_ns,
+                nonce,
+                value
```

Comment:
It's missing here `error_log`

Reply:
did you meant this line 
debug!(
                    "Receipt for allocation {} and signer {} failed reason: {}",
                    allocation_id.encode_hex(),
                    receipt_signer.encode_hex(),
                    receipt_error
            );

Reply:
We want to store the error logs for all receipts as well.

Take a look at the older version, we were inserting `error_log` in the old sql.